content/guides/guides/addressing.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Addressing in Browsers
+menu:
+    guides:
+        parent: guides
+weight: 2
+---
+
+<!--
+Based on the existing spec at:
+https://github.com/ipfs/in-web-browsers/blob/2896b7a642011bc92493971232b146e04c9a889c/ADDRESSING.md
+
+Note there are pending tasks to update/extend spec:
+https://github.com/ipfs/in-web-browsers/issues/147
+-->
+
+This document is a guide on how to address IPFS content paths on the web.
+
+## HTTP Gateways
+
+Gateways are provided strictly for convenience to help tools that speak HTTP but do not speak distributed protocols such as IPFS. They are the first stage on the upgrade path for the web.
+
+### Centralization Concerns
+
+HTTP Gateways has been working well since 2015 but comes with a significant set of limitations related to the centralized nature of HTTP and some of its semantics. Location-based addressing of a gateway depends on both DNS and HTTPS/TLS, which relies on the trust in Certificate Authorities and PKI. In the long term these issues should be mitigated by use of opportunistic protocol upgrade schemes.
```

Comment:
"HTTP gateways have worked well since 2015, but they come with a significant set of limitations related both to the centralized nature of HTTP and some of HTTP's semantics."
"which relies on a trust in certificate authorities"
"long term these" -> "long term, these"